Esri/arcgis-maps-sdk-swift-samples
# [Fix] Some Task block bugs

## Description

This PR fixes some button `Task` block related bugs:
- `Create load report`: Prevented "Run" presses when there is a create load report task already running.
- `Edit feature attachments`: Prevented fetch attachment data button presses when it has already been fetched successfully.
- `Find closest facility from point`: Prevented "Solve Routes" presses when there is a solve routes task already running.
- `Run valve isolation trace`: Prevented "Reset" presses when there is a trace task running as it can lead inconsistent state.
- `Show service area`: Prevented "Service Area" presses when there is a solve service area task already running.

## Linked Issue(s)

Related to: #316

## How To Test

- Press the buttons listed above repeatedly to verify that they no longer cause unexpected results.
- Ensure the samples still work as expected.
